packages/stark-ui/karma.conf.typescript.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Look in stark-testing for karma.conf.typescript.js
+ */
+
+const helpers = require("./node_modules/@nationalbankbelgium/stark-testing/helpers");
+
+let loadedConfiguration = null;
+
+// this object is just useful to retrieve the karma configuration object returned by our default Karma config
+// we need this since the karma configuration returns a function and not an object
+let configExtractor = {
+	set: configuration => {
+		loadedConfiguration = configuration;
+	}
+};
+
+/**
+ * Look in stark-testing for karma.conf.typescript.ci.js
+ */
+let defaultKarmaCIConfig = require("./node_modules/@nationalbankbelgium/stark-testing/karma.conf.typescript.js");
+
+// get the configuration object out
+defaultKarmaCIConfig(configExtractor);
+
+// start customizing the KarmaCI configuration from stark-testing
+let starkUiSpecificConfiguration = loadedConfiguration;
+
+if (!starkUiSpecificConfiguration.karmaTypescriptConfig.bundlerOptions.resolve) {
+	starkUiSpecificConfiguration.karmaTypescriptConfig.bundlerOptions.resolve = {};
+}
+
+// change the module resolution for the KarmaTypescript bundler
+starkUiSpecificConfiguration.karmaTypescriptConfig.bundlerOptions.resolve = Object.assign(starkUiSpecificConfiguration.karmaTypescriptConfig.bundlerOptions.resolve,
+	{
+		alias: {
```

Comment:
Could we maybe share this alias config between both karma config instead of duplicate it ?

Reply:
I'll leave it as it is for the moment because that requires some refactoring but I will create another issue to tack this ;)